shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Delegation/Certificates.hs

```diff
@@ -124,12 +122,32 @@ isRetirePool _ = False
 
 newtype PoolDistr crypto = PoolDistr
   { unPoolDistr ::
-      ( Map
-          (KeyHash 'StakePool crypto)
-          (Rational, Hash crypto (VerKeyVRF crypto))
-      )
+      Map (KeyHash 'StakePool crypto) (PoolStake crypto)
   }
-  deriving (Show, Eq, ToCBOR, FromCBOR, NFData, NoUnexpectedThunks, Relation)
+  deriving stock (Show, Eq)
+  deriving newtype (ToCBOR, FromCBOR, NFData, NoUnexpectedThunks, Relation)
+
+data PoolStake crypto = PoolStake
+  { poolStakeStake :: !Rational,
```

Comment:
Naming is hard, please help :flushed:. (There already exists a `poolStake` function).

Reply:
:laughing: . laughing with you, not at you, because indeed naming is hard!

how about `IndividualPoolStake` or some such?

Reply:
`IndividualPoolStake` sounds good, I'll make the change.